projects/step-frontend/src/lib/modules/timeseries/modules/legacy/components/execution-page/ts-execution-page.component.html

```diff
@@ -1,38 +1,37 @@
-<ng-container *ngIf="!executionHasToBeBuilt">
-  <div class="header-buttons">
-    <button
-      mat-icon-button
-      class="btn-settings grey-border"
-      [ngClass]="{ running: execution?.status === 'RUNNING' }"
-      #menuTrigger="matMenuTrigger"
-      [matMenuTriggerFor]="menu"
-    >
-      <step-icon name="settings" />
-    </button>
-  </div>
-  <mat-menu #menu="matMenu">
-    <step-ts-resolution-picker (resolutionChange)="handleResolutionChange($event)"></step-ts-resolution-picker>
-    <button mat-menu-item (click)="toggleCompareMode()">
-      <step-icon name="bar-chart-2" [class.active]="compareModeEnabled"></step-icon>
-      {{ compareModeEnabled ? 'Disable Compare Mode' : 'Enable Compare Mode' }}
-    </button>
-    <button mat-menu-item (click)="exportRawData()">
-      <step-icon name="download"></step-icon>
-      <span>Export Raw Measurements</span>
-    </button>
-  </mat-menu>
-  <step-timeseries-dashboard #dashboard *ngIf="dashboardSettings" [settings]="dashboardSettings">
-  </step-timeseries-dashboard>
-</ng-container>
-<div *ngIf="executionHasToBeBuilt" class="info-container">
-  <div class="content">
-    <div class="msg">
-      This execution has no time-series created yet. Start the building process clicking the button below.
+@if (!executionHasToBeBuilt) {
+  <!--  <div class="header-buttons">-->
+  <!--    <button-->
+  <!--      mat-icon-button-->
+  <!--      class="btn-settings grey-border"-->
+  <!--      [ngClass]="{ running: execution?.status === 'RUNNING' }"-->
+  <!--      #menuTrigger="matMenuTrigger"-->
+  <!--      [matMenuTriggerFor]="menu"-->
+  <!--    >-->
+  <!--      <step-icon name="settings"/>-->
+  <!--    </button>-->
+  <!--  </div>-->
+  <!--  <mat-menu #menu="matMenu">-->
+  <!--    <step-ts-resolution-picker (resolutionChange)="handleResolutionChange($event)"></step-ts-resolution-picker>-->
+  <!--    <button mat-menu-item (click)="toggleCompareMode()">-->
+  <!--      <step-icon name="bar-chart-2" [class.active]="compareModeEnabled"></step-icon>-->
+  <!--      {{ compareModeEnabled ? 'Disable Compare Mode' : 'Enable Compare Mode' }}-->
+  <!--    </button>-->
+  <!--    <button mat-menu-item (click)="exportRawData()">-->
+  <!--      <step-icon name="download"></step-icon>-->
+  <!--      <span>Export Raw Measurements</span>-->
+  <!--    </button>-->
+  <!--  </mat-menu>-->
```

Comment:
Is this part relevant?

Reply:
no, it's legacy. will be completely removed